pyramid_jinja2/__init__.py

```diff
@@ -1,28 +1,22 @@
 import inspect
-import os
-import posixpath
-import sys
-
 from jinja2 import Environment as _Jinja2Environment
-
 from jinja2.exceptions import TemplateNotFound
 from jinja2.loaders import FileSystemLoader
 from jinja2.utils import open_if_exists
-
+import os
+import posixpath
 from pyramid.asset import abspath_from_asset_spec
 from pyramid.path import DottedNameResolver
-
+import sys
```

Comment:
Something ain't right. isort or black should not be merging standard library imports with package imports. Was this a manual sort?

Reply:
I 100% agree with your comment. This is not the way I would sort this. BUT, once again, I used pyramid's isort config and this is the way imports are sorted there. Here for example: https://github.com/Pylons/pyramid/blob/master/src/pyramid/view.py

Reply:
Ah, yes, that's right. There may have been a reason for Pyramid's isort config. I remember it was driving me nuts in Deform when I used it as a model. I changed it to make more sense to me and to get black and isort to play nice with each other.

https://github.com/Pylons/deform/blob/main/pyproject.toml

Reply:
In Pyramid we made the decision to only treat imports within the project different, everything else is marked 3rd party. This is on purpose, @mmerickel may have more info on why.